index.html

```diff
@@ -596,6 +596,43 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pl'>
+
+				<p>Jesteśmy społecznością osób, którym nie jest obojętny rozwój oraz stan światowej sieci internetowej oraz wyrażąmy <strong>głęboki niepokój</strong> w związku z technologią <a href='https://ampproject.org/'>Accelerated Mobile Pages</a> („AMP” – przyspieszone mobilne strony internetowe). AMP to projekt, którzy rzekomo ma na celu poprawę wygody użytkowników podczas korzystania z sieci internetowej.</p>
+				
+				<p>Tymczasem AMP zatrzymuje użytkowników wewnątrz domeny Google oraz odciąga ruch z innych stron internetowych na korzyść Google. W skali miliardów użytkowników jest to działanie, które <strong>jeszcze bardziej umocni monopolistyczną pozycję Google w sieci internetowej</strong>.</p>
+				
+				<p>Zdajemy sobie sprawę z problemu powolnego ładowania się witryn internetowych w stosunku do własnościowych rozwiązań takich jak <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> i <a href='https://developer.apple.com/news-publisher/'>Apple News</a>. Wydawcy treści (zwłaszcza w sektorze mediów informacyjnych) od dawna zmagają się z trudnymi wyborami i niesłusznymi pobudkami, to prowadzi do złych decyzji i kompromisów, a te z kolei skutkują pogorszeniem wygody użytkowników.</p>
+				
+				<p>Silniki wyszukiwania to potężne narzędzia, dzięki którym można rozwiązać ten problem. Jednakże Google zdecydowało stworzyć specjalną pozycję na samej górze wyników wyszukiwania (dla artykułów) oraz dodaje specjalną ikonę <em>błyskawicy</em> (dla pozostałych treści), ale tylko dla wydawców korzystających z technologii kontrolowanej przez Google, udostępnianej przez Google w ramach ich infrastruktury, w ramach adresu URL Google oraz wewnątrz interfejsu stworzonego przez Google. (<a href='https://developers.google.com/search/docs/guides/about-amp'>źródło</a>)</p>
+				
+				<p>Idea AMP nie jest sama w sobie problematyczna, ale dwa aspekty jej implementacji umacniają Google na pozycji <em>de facto</em> standardowej platformy udostępniającej treść w sieci internetowej, a sam Google próbuje zwiększyć użycie AMP:</p>
+				
+				<ol>
+					<li>Treść, która dostępna jest wewnątrz AMP (a tym samym wewnątrz domeny Google), otrzymuje dodatkową promocję w ramach wyników wyszukiwania, w tym: najwyższą pozycję dla artykułów informacyjnych.</li>
+					<li>Kiedy użytkownik decyduje się na odwiedzenie witryny z treścią promowaną przez Google, bezwolnie pozostaje wewnątrz ekosystemu Google.</li>
+
+				</ol>
+				
+				<p>Jeżeli poprawa wygody użytkowników sieci internetowej jest prawdziwym celem AMP, pragniemy zaproponować kilka prostych zmian, które na to pozwolą, a jednocześnie nie zagrożą dynamice i konkurencyjności sieci internetowej i pozostawią ją skierowaną przede wszystkim na potrzeby konsumenta:</p>
```

Comment:
> pozostawią ją skierowaną przede wszystkim na potrzeby konsumenta

Maybe:

> zapewnią, że pozostanie skierowana przede wszystkim na potrzeby użytkownika

I would prefer term "użytkownik" over "konsument" in a context of Internet.

Reply:
Yes,  using _użytkownik_ here will also be consistent with the rest of the translation.